src/screens/Albums/HomeScreen.js

```diff
@@ -24,7 +24,7 @@ class HomeScreen extends Component {
             <Button title={item.title}
```

Comment:
* Inline Style is not clean and style should be in a separate file

Reply:
Done.